src/main/java/core/basesyntax/service/ReportGenerator.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.util.List;
+import java.util.Map;
+
+public interface ReportGenerator {
+    List<String> generateReport(Map<String, Integer> fruitStore);
```

Comment:
Pass BD to constructor
```suggestion
    List<String> generateReport();
```